### PR TITLE
added DigitalOceans Currents Survey

### PR DESCRIPTION
**Pull request by jnnck**
Open Source 2019

- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

Added the Digital Oceans Survey to get a Sammy Sticker, mentioned in issue #304

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -1,4 +1,13 @@
 [
+  {
+    "name": "DigitalOcean",
```
Comment (aslafy-z) [inline data.json]:
```suggestion
    "name": "Digital Ocean",
```

Comment (aslafy-z) [inline data.json]: This item has to be moved to be ordered alphabetically. Thanks

Comment (swapagarwal) [reply data.json]: @vikaspotluri123 Let's add it to the CI tests?

Comment (aslafy-z) [reply data.json]: Created an issue for it: https://github.com/swapagarwal/swag-for-dev/issues/334

**Comment by jnnck** [comment]
Didn't notice the alphabetical order, my excuses!

Updated pull request to reflect the 2 remarks above!

File: data.json [hunk 2]
```diff
@@ -89,6 +89,15 @@
     "dateAdded": "2018-02-18T07:48:58.000Z",
     "tags": ["clothing", "hacktoberfest", "expired", "stickers"]
   },
+  {
+    "name": "Digital Ocean",
+    "difficulty": "easy",
+    "description": "Complete this survey to receive a limited edition open source 'Sammy Shark' sticker!",
+    "reference": "https://digitalocean.getfeedback.com/r/p44ADlqG/",
+    "image": "https://pbs.twimg.com/media/EExmj5fXoAAnF9B?format=jpg&name=medium",
+    "dateAdded": "2019-10-01T11:00:00.000Z",
```
Comment (aslafy-z) [inline data.json]: We use the date of issue creation as dateAdded.

```suggestion
    "dateAdded": "2019-08-30T10:39:37.000Z",
```

Comment (swapagarwal) [reply data.json]: It's not allowing me to commit suggestion. Weird!
`You do not have permission to push to this repository.`

Comment (aslafy-z) [reply data.json]: image

I guess @jnnck unchecked this box

**Comment by jnnck** [comment]
Trying to figure out how i could check if i disabled... Sorry for the hassle! I'm new to this! 😄

**Comment by aslafy-z** [comment]
Created an issue for the blocking issue (swag reference links  are dead) : #340

**Comment by swapagarwal** [comment]
This survey has ended.

**Pull request closed without merging**